tgui/packages/tgui/interfaces/Sentencing.js

```diff
@@ -0,0 +1,362 @@
+import { useBackend, useLocalState } from '../backend';
+import { Flex, LabeledList, Box, Section, Tabs, Button } from '../components';
+import { Window } from '../layouts';
+
+const PAGES = {
+  'main': () => MainMenu,
+  'incident_report': () => NewReport,
+  'new_charge': () => NewCharge,
+};
+
+export const Sentencing = (props, context) => {
+  const { data } = useBackend(context);
+  const { current_menu } = data;
+  const PageComponent = PAGES[current_menu]();
+
+  return (
+    <Window theme="weyland" width={780} height={725}>
+      <Window.Content scrollable>
+        <PageComponent />
+      </Window.Content>
+    </Window>
+  );
+};
+
+const MainMenu = (props, context) => {
+  const { act } = useBackend(context);
+
+  return (
+    <Flex
+      direction="column"
+      justify="center"
+      align="center"
+      height="100%"
+      color="darkgrey"
+      fontSize="2rem"
+      mt="-3rem"
+      bold>
+      <Box fontFamily="monospace">Jurisdictional Automated System</Box>
+      <Box mb="2rem" fontFamily="monospace">
+        WY-DOS Executive
+      </Box>
+      <Box fontFamily="monospace">Version 5.8.4</Box>
+      <Box fontFamily="monospace">Copyright © 2182, Weyland Yutani Corp.</Box>
+
+      <Button
+        content="New Report"
+        width="30vw"
+        textAlign="center"
+        fontSize="1.5rem"
+        p="1rem"
+        mt="5rem"
+        onClick={() => act('new_report')}
+      />
+      <Box fontSize="2rem" mt="1rem">
+        OR
+      </Box>
+      <Box fontSize="1.5rem" mt="1rem">
+        scan an existing report
+      </Box>
+    </Flex>
+  );
+};
+
+const NewReport = (props, context) => {
+  const { data, act } = useBackend(context);
+  const { suspect_name, summary, sentence, current_charges } = data;
+  const canExport = suspect_name && current_charges.length;
+
+  return (
+    <>
+      <Flex mb=".25rem" align="center">
+        <Button.Confirm
+          icon="arrow-left"
+          px="2rem"
+          textAlign="center"
+          mr="1rem"
+          onClick={() => act('scrap_report')}
+        />
+
+        <h1>Incident Report</h1>
+
+        <Button
+          content="Export"
+          icon="print"
+          ml="auto"
+          px="2rem"
+          bold
+          tooltip={canExport ? '' : 'Missing suspect or charges'}
+          disabled={!canExport}
+          onClick={() => act('export')}
+        />
+      </Flex>
```

Comment:
I would definitely wrap this in a section tag for consistency, doesn't need a title